kernel-default.properties

```diff
@@ -205,8 +199,8 @@ keymanager_database_url = jdbc:postgresql://${mosip.kernel.database.hostname}:${
 keymanager_database_password=${db.dbuser.password}
 keymanager_database_username= keymgruser
 
-otpmanager_database_username = kerneluser
-otpmanager_database_url = jdbc:postgresql://${mosip.kernel.database.hostname}:${mosip.kernel.database.port}/mosip_kernel
+otpmanager_database_username=otpuser
```

Comment:
@ckm007 @Rakshitha650 - is this change valid?

Reply:
@ase-101 for mosip_otp db otpuser is valid but not for mosip_kernel db

Reply:
since otpmanager moved to a different db, this change is valid and resolving this comment